modules/ROOT/pages/swift.adoc

```diff
@@ -1068,9 +1069,15 @@ The Couchbase Lite API is thread safe except for calls to mutable objects: `Muta
 
 == Release Notes
 
+===  2.6.4 -- February 2020
+
+This maintenance release includes the following important fix:
+
+- https://issues.couchbase.com/browse/CBSE-7581[CBSE-7581] Fixes memory leak, preventing crashes
+
 === 2.6.3
 
-*Fixed in this Release*
+This maintenance release includes the following important fixes:
```

Comment:
Would avoid changing the format in a maintenance release. 
For 2.7.x releases, you could add a sub-heading for fixed issues (==== Fixed Issues) which is similar to the Server release notes.

Reply:
Thanks @amarantha-k . I've corrected these issues in latest commit  f94b921977b93ab711abf96dfc87bbfa34794718, which uses CBL-540 and CBL-619 in place of CBSE refs.